be/CMakeLists.txt

```diff
@@ -213,8 +213,8 @@ set_target_properties(openssl PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib
 add_library(leveldb STATIC IMPORTED)
 set_target_properties(leveldb PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libleveldb.a)
 
-add_library(jemalloc STATIC IMPORTED)
-set_target_properties(jemalloc PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib64/libjemalloc.a)
+add_library(jemalloc_for_arrow STATIC IMPORTED)
```

Comment:
Why not consider use a `--use-jemalloc` to optionally use jemalloc ?

Reply:
> Why not consider use a `--use-jemalloc` to optionally use jemalloc ?

I will do this, but not in this PR…there are a lot of things to adapt to using jemalloc, such as mem_hook and some mertics